actor/src/main/scala/org/apache/pekko/dispatch/VirtualThreadSupport.scala

```diff
@@ -85,15 +86,14 @@ private[dispatch] object VirtualThreadSupport {
       val ofVirtualClass = ClassLoader.getSystemClassLoader.loadClass("java.lang.Thread$Builder$OfVirtual")
       val ofVirtualMethod = classOf[Thread].getDeclaredMethod("ofVirtual")
       var builder = ofVirtualMethod.invoke(null)
-      if (executor != null) {
+      if (executor ne null) {
```

Comment:
Does this really make any difference?

Reply:
Yes，it was a function call.but now just a bytecode